integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -861,9 +861,11 @@ private[sql] case class CreateTable(cm: tableModel) extends RunnableCommand {
       val tablePath = catalog.createTableFromThrift(tableInfo, dbName, tbName, null)(sqlContext)
       try {
         sqlContext.sql(
-          s"""CREATE TABLE $dbName.$tbName USING carbondata""" +
-          s""" OPTIONS (tableName "$dbName.$tbName", tablePath "$tablePath") """)
-              .collect
+          s"""
+             | CREATE TABLE $dbName.$tbName
+             | USING carbondata
+             | OPTIONS (path "$tablePath")
```

Comment:
Ok, I think we should not keep both approach neither. Then I think we can keep the current approach. So shall I close this PR?